spec/requests/user_pages_spec.rb

```diff
@@ -56,4 +56,33 @@
 	  it { should have_title(user.name) }
 	end
 
+  describe "edit" do
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.